src/cli/cli.cpp

```diff
@@ -4097,17 +4097,50 @@ template <> otError Interpreter::Process<Cmd("pskc")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     otPskc  pskc;
 
+    /**
+     *@cli pskc
+     *@code
+     *pskc
+     *00000000000000000000000000000000
+     *Done
+     *@endcode
+     *@par api_copy
+     *#otThreadGetPskc
+     */
     if (aArgs[0].IsEmpty())
     {
         otThreadGetPskc(GetInstancePtr(), &pskc);
         OutputBytesLine(pskc.m8);
     }
     else
+
+    /**
+     *@cli pskc <key>
```

Comment:
We can't use <key> in the command name here, it's a parameter.  Instead let's do

`@cli pskc (set)`